geomesa-lambda/geomesa-lambda-datastore/pom.xml

```diff
@@ -119,6 +119,15 @@
             <groupId>org.specs2</groupId>
             <artifactId>specs2-junit_${scala.binary.version}</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-accumulo-datastore_${scala.binary.version}</artifactId>
```

Comment:
How was there not a dependency on GeoMesa Accumulo before?!

Reply:
this is the 'tests' classifier again, needed for the mini cluster wrapper class